### DISCUSSION TITLE
Technical issue with SETUP

### DISCUSSION THREAD

**Discussion by OndrejMottl** [CUNI-SPRouT/SPROuT-Forum#5] [IT help]
Do you have any issues with setting up your PC?

**Comment by Bryan-Novio** [comment]


**Reply by OndrejMottl** [reply]
Do you need help with something?

**Reply by Bryan-Novio** [reply]
oh, it's all okay for me. I just I thought we needed to mandatorily comment our name here:)

**Reply by OndrejMottl** [reply]
Your name should be written [here](https://github.com/orgs/CUNI-SPRouT/discussions/10)
